sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/rx/PermissionCrudTest.java

```diff
@@ -76,7 +80,7 @@ public void readPermission() throws Exception {
         CosmosResponseValidator<CosmosPermissionResponse> validator = new CosmosResponseValidator.Builder<CosmosPermissionResponse>()
                 .withId(permissionSettings.getId())
                 .withPermissionMode(PermissionMode.READ)
-                .withPermissionResourceLink("dbs/AQAAAA==/colls/AQAAAJ0fgTc=")
+                .withPermissionContainerName("AQAAAJ0fgTc=")
```

Comment:
this is called withPermissionContainerName but in the test `. withPermissionContainerName("AQAAAJ0fgTc=")` uses portion of the selflink.
So is this the container name or just a portion of the selflink?

ditto